schema/swagger-extensions.json

```diff
@@ -82,9 +82,6 @@
     },
     "externalDocs": {
       "$ref": "#/definitions/externalDocs"
-    },
-    "x-ms-secondary-file": {
```

Comment:
Wasn't being used anywhere?

Reply:
huh. Pulled the file from `master`, wasn't in `master` Weird.